deltacat/compute/compactor_v2/utils/task_options.py

```diff
@@ -49,10 +49,7 @@ def _calculate_parquet_column_size(
             if column_meta.path_in_schema in columns:
                 columns_found += 1
                 column_size += column_meta.total_uncompressed_size
-        assert columns_found == len(columns), (
```

Comment:
The delete memory needs to be accounted for in the merge tasks memory estimation as well.